integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -1094,23 +1094,105 @@ describe('Service provider referrals dashboard', () => {
 
     describe('with valid inputs and an appointment in the past', () => {
       describe('when booking for a phone call', () => {
-        it('presents an error because appointments in the past are not allowed', () => {
-          cy.visit(`/service-provider/action-plan/${actionPlan.id}/sessions/1/edit/start`)
+        describe('earlier today', () => {
+          it('takes the user through the feedback form upon appointment confirmation', () => {
+            cy.visit(`/service-provider/action-plan/${actionPlan.id}/sessions/1/edit/start`)
 
-          // schedule page
-          cy.get('#date-day').type('24')
-          cy.get('#date-month').type('3')
-          cy.get('#date-year').type('2021')
-          cy.get('#time-hour').type('9')
-          cy.get('#time-minute').type('02')
-          cy.get('#time-part-of-day').select('AM')
-          cy.get('#duration-hours').type('1')
-          cy.get('#duration-minutes').type('15')
-          cy.contains('1:1').click()
-          cy.contains('Phone call').click()
+            // schedule page
+            const today = new Date()
+            cy.get('#date-day').type(today.getDate().toString())
+            cy.get('#date-month').type((today.getMonth() + 1).toString())
+            cy.get('#date-year').type(today.getFullYear().toString())
+            cy.get('#time-hour').type('0')
```

Comment:
Should this be `12` (am) for midnight? I'm confused

Reply:
I just looked on the front end form. You cannot put 12:xx AM. You must put 0:xx AM.
For PM, you have to put 12:XX and not 0:XX though. (That may be obvious, I don't know)

Reply:
👍 Thanks Ryan. That sounds like an unconventional user experience and a potential bug in the picker code (12 am is definitely used in the world) -- but it overall doesn't matter anyway. No one books midnight appointments